### ISSUE TITLE
Autistic cell indexing...

### ISSUE THREAD

**Issue by Zeale** [Zeale/Krow#85]
There needs to be a fix label...

Anyways, the ScheduleModule's checkable cells use their own "getIndex()" method to get their indexes and retrieve their values. This didn't seem to be a problem for me (because it wasn't) until I got enough schedule tasks for me to need to scroll down to see the bottom ones. These bottom tasks in the module had checkable cells that were synced with the top ones. So if I selected "complete" on the top item, the bottom one would become completed. I wouldn't see this in action, because, by the nature of the bug, the two items must be separated by other items to the point that they aren't both visible on the screen at the same time.

Anyways, a TableRow's getIndex method() **and** a TableCell's getIndex() method don't show the actual index of the item in the table. That is expected to be obtainable from the list that backs the table. You know, using indexof() or something with the row's item. Anyways, the getIndex() method returns a value of the index of the cell as is assigned and reused for TableRows. So when you scroll past the first item to see the last item, the last item (which is now visible) has the index of the first item. Or something similar.

Long story short, getIndex() shouldn't be used where it is being used in the schedule module.

I already fixed this issue, but I'm writing this stuff all out for documentation purposes etc. I'll push the fix commit as soon as I'm done with this.
